WORKSPACE

```diff
@@ -72,12 +72,10 @@ node_repositories(
   yarn_version = "1.9.4",
 )
 
-# Use Bazel managed node modules. See more below: 
-# https://github.com/bazelbuild/rules_nodejs#bazel-managed-vs-self-managed-dependencies
 yarn_install(
   name = "npm",
-  package_json = "//:package.json",
-  yarn_lock = "//:yarn.lock",
+  package_json = "//tools:npm/package.json",
+  yarn_lock = "//tools:npm/yarn.lock",
```

Comment:
I'm not sure why we need this (I read something about RxJS). 

Why do we now need to this to compile RxJS? I'd like to avoid having another `package.json` file.

Reply:
It's a work-around for the time being since we're still building RXJS from source and rxjs uses `ts_library` which by default depends on the `@npm` workspace. This should be resolved soon. Its on the current tools sprint (TOOL-459).

Reply:
I see, please add a comment here and consider removing the lock file by locking the deps explicitly in the `package.json`.

Reply:
Sounds good. Added a comment and a TODO. The yarn.lock file is mandatory for `yarn_install` so will have to keep it.